.size-limit.json

```diff
@@ -19,4 +19,4 @@
     "limit": "2 kB",
     "gzip": true
   }
-]
+]
```

Comment:
There are still unrelated changes, please revert all of them

Reply:
I undid the changes in the .size-limit.json file and changed the addCrossorigin parameter from a boolean to a “crossorigin” string, so as to reduce the space during minification, as the user will be entering the value of the property.
Thank you.
Giorgio